crud/stats/init.lua

```diff
@@ -250,15 +250,23 @@ function stats.get(space_name)
 end
 
 local function resolve_space_name(space_id)
-    local replicasets = vshard.router.routeall()
+    -- Resolving name with custom vshard router is not supported.
+    local vshard_router = vshard.router.static
+
+    if vshard_router == nil then
+        log.warn('Failed to resolve space name for stats: default vshard router not found')
+        return nil
+    end
```

Comment:
Could we add a `vshard_router` option for stats.wrap?

Reply:
Since this case is related only to `crud.len`, we can make specific wrappers for it. But this behavior is deprecated and will be removed, so I decided not to make statistics code more complicated since it will become irrelevant after a couple of releases.

Reply:
`Nothing is more permanent than a temporary solution.`

The inconsistency of the code is not good, but you know more about the plans for the development of this product. So it's up to you.